### ISSUE TITLE
Compile qr_mumps 3.1.0 with Homebrew and Yggdrasil

### ISSUE THREAD

**Issue by amontoison** [JuliaSmoothOptimizers/QRMumps.jl#90]
cc @dpo 
It will fix the issue with CI builds that depends on the Homebrew version.

**Comment by dpo** [comment]
Do you mean that the qr_mumps formula should be updated in the mumps_jl Homebrew tap?

**Comment by amontoison** [comment]
Yes, we should update the Homebrew tap in `dpo/mumps-jl` and MUMPS_jll.jl with Yggdrasil.
I quickly send an email to Alfredo this afternoon to ask what was modified with the new release.

**Comment by dpo** [comment]
Homebrew attempt: https://github.com/dpo/homebrew-mumps-jl/pull/21

**Comment by dpo** [comment]
There are new bottles for QRMumps 3.1 and MUMPS 5.5.1. I still have to upgrade MUMPS.
